torch/csrc/jit/codegen/cuda/arith.h

```diff
@@ -364,6 +364,8 @@ TORCH_CUDA_CU_API TensorView* expand(
 // non broadcasted iter domain, inp will be expanded to other's size.
 TORCH_CUDA_CU_API TensorView* expand_as(TensorView* inp, TensorView* other);
 
+TORCH_CUDA_CU_API std::vector<Val*> symbolic_sizes(TensorView* inp);
```

Comment:
Should we take as a guidance ATen C++ naming or PyTorch naming?
In PyTorch, the shape/size of a tensor is obtained with `tensor.shape` or `tensor.size()`.
In ATen C++ the shape/size of a tensor is obtained with `tensor.sizes()`.

Reply:
I changed it to `tensor_sizes()`.